sdk/servicebus/azure-servicebus/azure/servicebus/aio/_async_message.py

```diff
@@ -108,7 +108,7 @@ async def abandon(self):
 
     async def defer(self):
         # type: () -> None
-        """Abandon the message. This message will be returned to the queue to be reprocessed.
+        """Defers the message, to be received via ~azure.servicebus.ServiceBusReceiver.receive_deferred_messages by sequence number
```

Comment:
```suggestion
        """Defer the message, to be received via ~azure.servicebus.ServiceBusReceiver.receive_deferred_messages by sequence number
```

Reply:
we might want to apply the changes to sync message as well

Reply:
Per our discussion used the version from T1 with the key word I was trying to communicate being "Must" (be received via sequence number)